src/main/java/service/impl/TransactionDataImpl.java

```diff
@@ -0,0 +1,35 @@
+package service.impl;
+
+import java.util.List;
+import java.util.Map;
+import service.Strategy;
+import service.TransactionData;
+import service.Validator;
+
+public class TransactionDataImpl implements TransactionData {
```

Comment:
більш підходяща назва
```suggestion
public class FruitServiceImpl implements FileService {
```